### PR TITLE
Make use of musl's `weak` macro. NFC

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: tools/system_libs.py [hunk 1]
```diff
@@ -2047,6 +2047,7 @@ def can_use(self):
 class libstubs(DebugLibrary):
   name = 'libstubs'
   src_dir = 'system/lib/libc'
+  includes = ['system/lib/libc/musl/src/include']
```
Comment (kripken) [inline tools/system_libs.py:2050]: This PR doesn't add new `#include`s - is this addition necessary?

Comment (sbc100) [reply tools/system_libs.py:2050]: Its so that system/lib/libc/emscripten_syscall_stubs.c doesn't need to redefine `weak` that musl's features.h already provides.

Comment (kripken) [reply tools/system_libs.py:2050]: But how does that `features.h` get included, if there isn't a new `#include <features.h>` in this PR?

Comment (sbc100) [reply tools/system_libs.py:2050]: features.h is gets included transitively, and there is public and a private copy.  In this case just adding the internal include path gives access to musl-internal macros like this.

Comment (kripken) [reply tools/system_libs.py:2050]: I see, wow, that is surprising but I guess it make sense.

**Review by kripken — APPROVED** [review]
